tests/testthat/test-tidy-upkeep.R

```diff
@@ -12,6 +12,41 @@ test_that("upkeep bullets don't change accidentally", {
   create_local_package()
 
   expect_snapshot(writeLines(
-    upkeep_checklist(posit_pkg = TRUE, posit_person_ok = FALSE)
+    tidy_upkeep_checklist(posit_pkg = TRUE, posit_person_ok = FALSE)
   ))
 })
+
+test_that("upkeep bullets don't change accidentally",{
+  skip_if_no_git_user()
+  withr::local_options(usethis.description = NULL)
+  create_local_package()
+
+  # Need to add git so can detect presence of master branch
+  use_git()
```

Comment:
Even if you stick with this (which I think we should not), you'd have to go to even more effort to make sure the default branch is `master`. Locally this fails for me because I have configured my default for the default branch to be `main`. More reason to design this away.